08-js-security/level2/index.secure.html

```diff
@@ -0,0 +1,91 @@
+ecu<!doctype html>
```

Comment:
is this some kind of a s**ecu**re prefix for doctype?

Reply:
Level2 doesn't seem to work for me:

![image](https://user-images.githubusercontent.com/16776066/61909359-9ee0b000-af3a-11e9-9212-cdd0a4291f2a.png)

Am I doing something wrong?

Reply:
OK, this only happens when you first save a problematic message while using a non-secure version, and then load it using the secure one.
Fix this, everything else looks nice.